mock-web-services/apibridge/__init__.py

```diff
@@ -0,0 +1,136 @@
+"""
+Mock service for testing the API Bridge
+"""
+
+import time
+
+from apibridge import sample_data
+from flask import Blueprint, Response, jsonify, request
+
+from .models import (
+    BridgeResponse,
+    CompatibilityLevel,
+    DiscoveryResponse,
+    Endpoint,
+    HealthcheckResponse,
+    ProblemDetail,
+    ValidationProblemDetail,
+)
+
+apibridge_blueprint = Blueprint("apibridge", __name__)
+
+
+@apibridge_blueprint.route("/health-check")
+def healthcheck() -> tuple[Response, int]:
+    """Health check endpoint to verify service status"""
+    try:
+        response_code = int(request.headers.get("Emulate-Response-Code", 200))
+
+        match response_code:
+            case 400 | 401 | 429 | 500:
```

Comment:
It will be good to filp it may be? case 200:  proceed case _: create_problem_detail()

Reply:
200 is the default/fallback which needs to be last and will only happen if there are no errors. The other response codes in the `match` are there only to be used if there is an explicit header forcing emulation of the other supported http codes.